package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "babel-preset-enact",
+  "version": "1.0.0",
+  "description": "Babel preset used by Enact",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/enactjs/babel-preset-enact.git"
+  },
+  "homepage": "https://enactjs.com",
+  "license": "Apache-2.0",
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
+    "@babel/plugin-proposal-decorators": "^7.16.4",
+    "@babel/plugin-proposal-export-default-from": "7.18.9",
```

Comment:
Why some of dependencies were pinned?

Reply:
I think it's copy/paste problem. I will fix them.